app/scripts/controllers/moodCatAudio.js

```diff
@@ -73,7 +77,33 @@
        }
      });
 
-     $scope.loadSong(track);
+     $scope.sendClassification = function sendClassification() {
+         if(!angular.isObject($scope.sound)) {
+             return;
+         }
+
+         var arousal = $("[name='arousalFeedback']:checked");
+         var valence = $("[name='valenceFeedback']:checked");
```

Comment:
If using `ng-model` as suggested above, you can use the ng-model bound variable from the current `$scope` instead. This is the jQuery way of doing this, but it makes it untestable, as it requires a properly formed DOM with elements. Using `$scope` and `ng-model` would be the AngularJS way :wink: